src/aspire/utils/rotation.py

```diff
@@ -234,10 +234,29 @@ def from_euler(values, dtype=np.float32):
         :param values: Rotation angles in radians, as a n x 3 array
         :return: new Rotation object
         """
-        rotations = sp_rot.from_euler("ZYZ", values.astype(dtype), degrees=False)
+        rotations = sp_rot.from_euler("ZYZ", values, degrees=False)
         matrices = rotations.as_matrix().astype(dtype)
         return Rotation(matrices)
 
+    @staticmethod
+    def about_axis(axis, angles, dtype=np.float32):
+        """
+        Build rotation object from axis and angles of rotation.
+
+        :param axis: A string denoting the axis of rotation. "x", "y", or "z".
+        :param angles: Rotation angles in radians.
```

Comment:
If I understand this correctly, since this is a rotation about a single axis, `angles` can only be a single value, or array-like with shape `(N)` or `(N,1)`. Maybe just do a check of the shape and indicate in the docstring what the allowed values are? This will also distinguish this method's behavior from our own `from_euler` method.
  (this is from https://docs.scipy.org/doc/scipy/reference/generated/scipy.spatial.transform.Rotation.from_euler.html#scipy.spatial.transform.Rotation.from_euler)

Wouldn't be obvious unless the user looks into the body of the method and then looks up SciPy's method.

Reply:
You are correct. I'll put in a shape check and update the doc string. Thanks.

Reply:
@chris-langfield After looking at this for a minute, I'm not sure checking the shape is necessary. Since we've enforced that the rotation is about a single axis, SciPy will check the shape and produce a meaningful `ValueError` if the shape is not as expected.

Reply:
But we're basically splitting the SciPy method's functionality between two of our own. IMO it makes more sense to catch it ourselves because we're enforcing the division. But we can just see what Joakim thinks

Reply:
Good point. I'll just add the logic and we can nix it later if need be.